csrc/scheduler/normalization_utils.cpp

```diff
@@ -841,7 +883,16 @@ bool projectBufferToInputs(
         expr->as<UnaryOp>()->getUnaryOpType() == UnaryOpType::Exp) {
       has_exp_op = true;
     }
+    if (expr->isA<RNGOp>()) {
```

Comment:
qq: why not return false directly here?
Same question for the logic in `has_exp_op`.

Reply:
changed to directly return false if has rng op.
For `has_exp_op`, it is not very expensive compared with rng op, we want to further consider the benefit of projecting to inputs, e.g. if the register pressure is high, we still want to project to inputs to save some registers at the cost of doing additional exp ops.

Reply:
got'ya. Sorry I wasn't paying enough attention in the logic of `has_exp_op`.